backend/models/UserModel.ts

```diff
@@ -1,30 +1,45 @@
 import mongoose from "mongoose";
+import bcrypt from "bcrypt";
 
 export interface IUser extends Document {
```

Comment:
Perhaps, it would be good to add a field: user_role. And add this information to the access token. Because, in my opinion, we will need this information in the future.

Reply:
We should discuss user roles in more detail with a team

Reply:
Ok